packages/ndla-ui/src/SearchTypeResult/index.js

```diff
@@ -7,13 +7,15 @@
  */
 
 import SearchTypeResult from './SearchTypeResult';
-import SearchTypeHeader from './SearchTypeHeader';
-import SearchTypeTopicFilter from './SearchTypeTopicFilter';
 import SearchHeader from './SearchHeader';
+import SearchFieldHeader from './SearchFieldHeader';
+import SearchNotionsResult from './SearchNotionsResult';
+import SearchSubjectResult from './SearchSubjectResult';
 
 export {
```

Comment:
Er det ikkje bedre å endre denne fila fra js til ts og heller ha den med i index.ts i stedet for index-javascript? Dei andre filene i mappa er vel tsx.

Reply:
Endra